openapi3/schema.go

```diff
@@ -1192,6 +1313,12 @@ func markSchemaErrorIndex(err error, index int) error {
 		v.reversePath = append(v.reversePath, strconv.FormatInt(int64(index), 10))
 		return v
 	}
+	if v, ok := err.(MultiError); ok { // TODO: this might need to be an indexed for loop
```

Comment:
Can this TODO be removed? What about it?

Reply:
Ah yes. I was able to validate this with a test and forgot to remove